.vsts-ci/templates/releaseBuildPhase.yml

```
@@ -12,32 +12,89 @@ jobs:
     vmImage: ${{ parameters.vmImage }}
 
   steps:
+
+  - powershell: |
+        sudo chown root:root /
+    displayName: 'Make sure root owns root'
+    condition: succeeded()
+
   - powershell: 'dir env:'
     displayName: 'Capture Environment'
 
+  - powershell: |
+        $channel = (get-content ./snapcraftConfig.json | ConvertFrom-Json).channel
+        Write-Verbose -Verbose -Message "using Channel $channel"
+        sudo snap install snapcraft --classic "--channel=$channel"
+    displayName: Install snapcraft
+    condition: succeeded()
+
   - powershell: |
       $releaseTagName = switch('$(channel)') {
         'stable' {
+          cd stable
           'StableReleaseTag'
         }
+        'lts' {
+          cd lts
+          'LtsReleaseTag'
+        }
         'preview' {
+          cd preview
           'PreviewReleaseTag'
         }
         default {
+          cd preview
           'PreviewReleaseTag'
         }
       }
       Write-Host "releaseTagName: $releaseTagName"
-      $releaseTag = Invoke-RestMethod -Uri 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json' | Select-Object -ExpandProperty $releaseTagName
+      $releaseTag = Invoke-RestMethod -Uri 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json' | Select-Object -ExpandProperty $releaseTagName -First 1
```

This can be simplified:

```suggestion
      $releaseTag = (Invoke-RestMethod -Uri 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json').$releaseTagName
```

---

I need to get the first one

---

```suggestion
      $releaseTag = @(Invoke-RestMethod -Uri 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json').$releaseTagName[0]
```

---

You really prefer this syntax?